ReactNativeClient/lib/models/Setting.js

```diff
@@ -8,6 +20,7 @@ const { toTitleCase } = require('lib/string-utils.js');
 const { rtrimSlashes } = require('lib/path-utils.js');
 const { _, supportedLocalesToLanguages, defaultLocale } = require('lib/locale.js');
 const { shim } = require('lib/shim');
+const { bridge } = require('electron').remote.require('./bridge');
```

Comment:
Oh sorry I missed that. We can't have a dependency to the Electron package here as it needs to compile in React Native too. I think it should work if you use `shim.openUrl` instead.

Reply:
Good catch! Thanks.